tests/functional/helper.js

```diff
@@ -419,6 +427,14 @@ module.exports = {
     _lastNotification.message = null;
   },
 
+  lastAddonLoadingCompleted() {
+    return _lastAddonLoadingCompleted;
+  },
+
+  resetLastAddonLoadingCompleted() {
+    _lastAddonLoadingCompleted = false;
+  },
+
```

Comment:
nit: These two are not necessary because lastAddonLoadingCompleted is not used outside of this module.